docs/docs/plugin-authoring.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Plugin Authoring
+---
+
+Creating custom plugins in Gatsby is a breeze with our straightforward authoring process.
+
+## Core Concepts
+
+- Each Gatsby plugin can be installed as an npm package or as a [local plugin](#local-plugins)
```

Comment:
Capitalize NPM

Reply:
Their official branding appears to keep it lowercase: https://www.npmjs.com/ 😕